Directory.Build.props

```diff
@@ -19,6 +19,7 @@
     </PropertyGroup>
 
     <PropertyGroup>
+        <NoWarn>S2696</NoWarn>
```

Comment:
If we really need to exclude that, I would go for the `.editorconfig`.

Reply:
Sure. I had to add that as it seemed like it was more of a SonarQube bug than a problem with the code. Or I just couldn't figure out how to change the code and make it happy